src/components/module-intro.tsx

```diff
@@ -0,0 +1,99 @@
+import Image from "next/image";
+import { Fragment } from "react";
+import Button from "@/components/button";
+import { ArrowLeft } from "@phosphor-icons/react";
+import { useRouter } from "next/navigation";
+
+export type ModuleChapter = {
+  icon: string;
+  title: string;
+  minutes?: string;
+};
+
+export type ModuleIntroProps = {
+  title: string;
+  description: string;
+  background?: string;
+  entryPath: string;
+  chapter: ModuleChapter[];
+};
+
+export default function ModuleIntro({
+  title,
+  description,
+  chapter,
+  background,
+  entryPath,
+}: ModuleIntroProps) {
+  const router = useRouter();
+  return (
+    <div className="h-full w-full flex flex-col items-start relative">
+      <div
+        className="flex items-center gap-4 mb-4 cursor-pointer transition-colors hover:text-sky-800"
+        onClick={() => router.push("/space")}
+      >
+        <ArrowLeft />
+        <span>Zurück zur Übersicht</span>
+      </div>
```

Comment:
brauchen wir das wirklich? Ich glaube der User wird das logo an klicken, um an die Startseite zu kommen. 

Code: div hat niemals ein onClick Function. Accessibility für Blinde. Nutze gerne  einen Link in next js

Reply:
Lol, logisch, Link is ja genau dafür da. Ersetz ich noch 👍

Ich glaube fast, dass die explitize Anzeige klarer ist, ein Logo ist keine klare Navigation. 

Weiß nicht, ob der Vergleich Sinn ergibt, im Material Design gabs lange diese Hamburger Menüs, ich behaupte die wurden deswegen größten Teils durch bottom navigations ausgetauscht, weil das Symbol nicht gut genug kommuniziert, dass es da eine Navigation gibt. Ich glaube, hier wäre es ähnlich, das Logo gibt nicht die Ansage "hiermit navigierst du".